src/_data/pricingFeatures.yaml

```diff
@@ -18,127 +23,122 @@ sections:
       - id: custom-hostname
         label: Custom Hostnames
-        cloudValues: [ null, null, 'check' ]
+        cloudValues: [ null, null, null, 'check' ]
         selfHostedValues: [ null, 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Access your Node-RED application via your own domain name.</p>"
       - id: persistent-files
         label: Persistent files
-        cloudValues: [ '1 GB', '10 GB', '100 GB' ]    
+        cloudValues: [ null, '1 GB', '10 GB', '100 GB' ]    
         tags: [ 'cloud' ]
         info: "<p>FlowFuse allows Node-RED instances to store files persistently to local file system. This is the space limit for these files.</p>"
       - id: persistent-context
         label: Persistent Context
-        cloudValues: [ '10 MB', '100 MB', '1 GB' ]    
+        cloudValues: [ null, '10 MB', '100 MB', '1 GB' ]    
         tags: [ 'cloud' ]
         info: "<p>In-memory values defined in a Node-RED flow will persist across project restarts and upgrades.</p>"
   - label: Scale
     rows:
       - id: device-mgmt
         label: Device Management
-        cloudValues: [ 'check', 'check', 'check' ]
+        cloudValues: [ 'check', 'check', 'check', 'check' ]
         selfHostedValues: [ 'check', 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Connect to edge devices to quickly assess and update logic. Debug one device and roll out improvements to your fleet in minutes. All securely without requiring full device access for your whole organisation.</p>"
       - id: edge-development
         label: Edge Development
-        cloudValues: [ null, 'check', 'check' ]
+        cloudValues: [ null, null, 'check', 'check' ]
         selfHostedValues: [ null, 'check' ]    
         tags: [ 'cloud', 'self-hosted' ]
         info: "<p>Connect to an edge device and develop your flows.</p>"
       - id: mqtt-broker-cloud
         label: MQTT Broker
-        cloudValues: [ null, 'check', 'check' ]    
+        cloudValues: [ null, 'check', 'check', 'check' ]    
         tags: [ 'cloud' ]
-        info: "<p>Easily manage and create MQTT clients to transport data for efficient messaging and communication within your applications. The Team Tier includes 5 clients at no extra cost, while the Enterprise Tier includes 20 clients for free as part of your existing plan, with purchase additional clients as needed in the future.</p>"
+        info: "<p>Easily manage and create MQTT clients to transport data for efficient messaging and communication within your applications. At no extra cost, the Starter Tier includes 2 clients, the Team Tier includes 5 clients, and the Enterprise Tier includes 20 clients as part of your existing plan. Additional clients can be purchased as needed in the future.</p>"

```

Comment:
Let's remove the at no extra cost part. That should be obvious from the table

Reply:
Done.